tests/network-tests/src/misc/postRuntimeUpgrade.ts

```diff
@@ -64,5 +64,9 @@ export default async function assertValues({ api }: FlowProps): Promise<void> {
   // assert.deepEqual(curatorGroup.permissionsByLevel, expectedPermissions)
   assert.equal(`${curatorGroup.permissionsByLevel}`, `${expectedPermissions}`)
 
+  // Damping factor after runtime should be 100% = default
```

Comment:
Some of the previous assertions in this function need to be removed or fixed, they were for ephesus -> nara upgrade. As well as the setup in the `preRuntimeUpgrade.ts`